WPFHospitalEditor/AdditionalInformation.xaml.cs

```diff
@@ -18,7 +17,7 @@ public partial class AdditionalInformation : Window
     {
         MapObjectController mapObjectController = new MapObjectController();
         EquipmentServerController equipmentServerController = new EquipmentServerController();
-        MedicationServerController medicationServerController = new MedicationServerController();
+        IMedicationServerController medicationServerController = new MedicationServerController();
```

Comment:
I guess you need to change other fields to interfaces.

Reply:
I didn't want to change it in this branch, because we will make branch 'refactor' or something where we will this.